Bio/SeqIO/InsdcIO.py

```diff
@@ -647,17 +659,20 @@ def _write_the_first_line(self, record):
             # just the generic Alphabet (default for fasta files)
             raise ValueError("Need a DNA, RNA or Protein alphabet")
 
+        topology = self._get_topology(record)
+
         division = self._get_data_division(record)
 
         assert len(units) == 2
         assert len(division) == 3
         # TODO - date
         # TODO - mol_type
-        line = "LOCUS       %s %s %s    %s           %s %s\n" \
+        line = "LOCUS       %s %s %s    %s  %s %s %s\n" \
```

Comment:
That's how I could make it align to the LOCUS line from the GenBank headers I looked at. I didn't find the exact definition for this in the spec. Then again, the spec is very confusing, which is the reason why we have that much fun with that format in the first place.